packages/core/src/components/slider/handle.tsx

```diff
@@ -153,9 +154,9 @@ export class Handle extends AbstractPureComponent<IHandleProps, IHandleState> {
     private handleMoveEndedAt = (clientPixel: number) => {
         this.removeDocumentEventListeners();
         this.setState({ isMoving: false });
-        // not using changeValue because we want to invoke the handler regardless of current prop value
+        // always invoke onRelease; changeValue may call onChange if value is different
```

Comment:
this is a small API improvement. was weird that _only_ onRelease was invoked in this case.